docroot/modules/custom/prisoner_hub_jsonapi_resources/prisoner_hub_jsonapi_resources.routing.yml

```diff
@@ -0,0 +1,6 @@
+prisoner_hub_jsonapi_resources.topics:
+  path: '/%jsonapi%/topics'
```

Comment:
Hmm I think I might change this.  As it's not strictly valid in the JSON:API spec.  Firstly it should be "topic" instead of "topics".  And if we have a resource called "topic", then to follow the spec, each individual resource should be available underneath this.  I.e. `/jsonapi/topic/{{uuid}}`.

The alternative approach is to enable the endpoint `jsonapi/taxonomy_term` (which can be done via the https://www.drupal.org/project/jsonapi_cross_bundles) module).  You could then use a `?filter` parameter to limit the items to be just categories and tags.  Unfortunately, the filter looks a little fiddly due to some weird quirk in Drupal, and having to use an `IN`.  The final url would look a little bit like this:
`jsonapi/prison/berwyn/taxonomy_term?filter[vid-filter][path]=type.meta.drupal_internal__target_id&filter[vid-filter][operator]=IN&filter[vid-filter][value][1]=moj_categories&filter[vid-filter][value][2]=tags`